pkgs/pillow.yaml

```diff
@@ -1,7 +1,21 @@
 extends: [setuptools_package]
 dependencies:
-  build: [png]
+  build: [zlib,png,freetype,openjpeg]
 
 sources:
 - key: tar.gz:q5qmcgfaefplufr7o6bbcdt67tn3cx4k
   url: https://pypi.python.org/packages/source/P/Pillow/Pillow-2.8.1.tar.gz
+
+build_stages:
+- name: install
+  mode: replace
+  handler: bash
+  bash: |
+       ${PYTHON} -c 'import setuptools; __file__="setup.py"; exec(open(__file__).read())' \
```

Comment:
This looks really messed up --- this hack should only happen in the `setuptools_package.yaml` I would think. I am ok with this as a temporary workaround to make it do what you need, but as a long term solution I would like to figure out a way to perhaps add some parameters to `setuptools_package.yaml` that you just set here. Doesn't just adding `libflags` into `extends` fix this issue?